netmiko/base_connection.py

```diff
@@ -534,6 +567,13 @@ def establish_connection(self, width=None, height=None):
         if self.protocol == 'telnet':
             self.remote_conn = telnetlib.Telnet(self.host, port=self.port, timeout=self.timeout)
             self.telnet_login()
+        elif self.protocol == 'serial':
+            self.serial_settings["port"] = str(self.port).split(" ")[0]
+            for k in self.serial_settings:
+                self.serial_defaults[k] = self.serial_settings[k]
+            self.serial_settings = self.serial_defaults
```

Comment:
I think we can just get rid of these lines and just use the `update` method in `__init__`.